.circleci/deploy-to-pantheon.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -ex
+
+TERMINUS_DOES_MULTIDEV_EXIST()
+{
+    # Stash list of Pantheon multidev environments
+    PANTHEON_MULTIDEV_LIST="$(terminus multidev:list -n ${TERMINUS_SITE} --format=list --field=Name)"
+
+    while read -r multiDev; do
+        if [[ "${multiDev}" == "$1" ]]
+        then
+            return 0;
+        fi
+    done <<< "$PANTHEON_MULTIDEV_LIST"
+
+    return 1;
+}
+
+if [[ $CIRCLE_BRANCH == "master" ]]
+then
+    terminus build:env:push -n "$TERMINUS_SITE.$TERMINUS_ENV" --yes
+    terminus secrets:set -n "$TERMINUS_SITE.$TERMINUS_ENV" token "$GITHUB_TOKEN" --file='github-secrets.json' --clear --skip-if-empty
```

Comment:
```
echo -e "CircleCI will only deploy to Pantheon for master or pull requests.\n"
exit 1;
```

I am probably going to change this to `exit 0;` so that the build does not fail if on a branch but instead just skips deployment. This way a passing/failing build on GitHub can be associated with passing/failing code sniffing and unit testing.